### DISCUSSION TITLE
Double licensing Theengs Decoder

### DISCUSSION THREAD

**Discussion by 1technophile** [theengs/decoder#435] [General]
Dear Theengs Decoder contributors,

Thanks for your participation in Theengs Decoder!
I have requests to use Theengs Decoder on closed-source applications or with a different license than GPL V3.

I would like to know if you are ok with a double licensing of Theengs Decoder, it means that the public repository will remain GPL V3 and third parties can contact me to have a distribution with another license if they are interested.

Please answer with yes or no, if no answer is given during November I will consider this as a yes.

@DigiH 
@h2zero 
@koenvervloesem 
@emericg 
@Cadsters 
@matthias-bs 
@realzhiqiang 
@kabili207 
@toomyem 
@alegsan 
@sebbo2002 
@hallard 
@adamrbell 
@mjeanrichard 
@bbacskay 
@mrbubble62

**Comment by DigiH** [comment]
Yes 👍

**Reply by mrbubble62** [reply]
Yes

**Comment by hallard** [comment]
Yes

Le 30 oct. 2023 à 14:49, Florian ***@***.***> a écrit :


Dear Theengs Decoder contributors,

Thanks for your participation in Theengs Decoder!
I have requests to use Theengs Decoder on closed-source applications or with a different license than GPL V3.

I would like to know if you are ok with a double licensing of Theengs Decoder, it means that the public repository will remain GPL V3 and third parties can contact me to have a distribution with another license if they are interested.

Please answer with yes or no, if no answer is given during November I will consider this as a yes.

@DigiH <https://github.com/DigiH>
@h2zero <https://github.com/h2zero>
@koenvervloesem <https://github.com/koenvervloesem>
@emericg <https://github.com/emericg>
@Cadsters <https://github.com/Cadsters>
@matthias-bs <https://github.com/matthias-bs>
@realzhiqiang <https://github.com/realzhiqiang>
@kabili207 <https://github.com/kabili207>
@toomyem <https://github.com/toomyem>
@alegsan <https://github.com/alegsan>
@sebbo2002 <https://github.com/sebbo2002>
@hallard <https://github.com/hallard>
@adamrbell <https://github.com/adamrbell>
@mjeanrichard <https://github.com/mjeanrichard>
@bbacskay <https://github.com/bbacskay>
@mrbubble62 <https://github.com/mrbubble62>

**Comment by emericg** [comment]
Of course :+1:

**Comment by realzhiqiang** [comment]
yes

**Comment by h2zero** [comment]
Yes

**Comment by koenvervloesem** [comment]
Yes

**Comment by sebbo2002** [comment]
Yes

**Comment by matthias-bs** [comment]
Yes, thanks for asking!

**Comment by Cadsters** [comment]
Yes

Van: Florian ***@***.***>
Verzonden: maandag 30 oktober 2023 14:49
Aan: theengs/decoder ***@***.***>
CC: Cadsters ***@***.***>; Mention ***@***.***>
Onderwerp: [theengs/decoder] Double licensing Theengs Decoder (Discussion #435)


Dear Theengs Decoder contributors,

Thanks for your participation in Theengs Decoder!
I have requests to use Theengs Decoder on closed-source applications or with a different license than GPL V3.

I would like to know if you are ok with a double licensing of Theengs Decoder, it means that the public repository will remain GPL V3 and third parties can contact me to have a distribution with another license if they are interested.

Please answer with yes or no, if no answer is given during November I will consider this as a yes.

@DigiH<https://github.com/DigiH>
@h2zero<https://github.com/h2zero>
@koenvervloesem<https://github.com/koenvervloesem>
@emericg<https://github.com/emericg>
@Cadsters<https://github.com/Cadsters>
@matthias-bs<https://github.com/matthias-bs>
@realzhiqiang<https://github.com/realzhiqiang>
@kabili207<https://github.com/kabili207>
@toomyem<https://github.com/toomyem>
@alegsan<https://github.com/alegsan>
@sebbo2002<https://github.com/sebbo2002>
@hallard<https://github.com/hallard>
@adamrbell<https://github.com/adamrbell>
@mjeanrichard<https://github.com/mjeanrichard>
@bbacskay<https://github.com/bbacskay>
@mrbubble62<https://github.com/mrbubble62>

**Comment by mjeanrichard** [comment]
Yes.

**Comment by kabili207** [comment]
Yes, although I am curious what closed-source-friendly licenses you've considered

**Reply by 1technophile** [reply]
Working on it, topic still open on my side.

**Comment by bbacskay** [comment]
Yes

**Comment by alegsan** [comment]
Yes

**Comment by adamrbell** [comment]
Yes

**Comment by toomyem** [comment]
Yes

**Comment by 1technophile** [comment]
Thanks for your prompt answers!
